dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -76,6 +75,24 @@ def needed_verifications(level: int) -> int:
     return _requirements[f"l{level}"]["nodesRequired"]
 
 
+def get_l5_wait_time(chain_id: str) -> int:
+    #  L5s should have an endpoint that tells that
+    network_attr = {
+        "btc": {"confirmations": btc.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": btc.AVERAGE_BLOCK_TIME, "delay_buffer": 1.5},
+        "eth": {"confirmations": eth.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": eth.AVERAGE_BLOCK_TIME, "delay_buffer": 1.5},
+        "bnb": {"confirmations": bnb.CONFIRMATIONS_CONSIDERED_FINAL, "block_time": bnb.AVERAGE_BLOCK_TIME, "delay_buffer": 1.5},
+    }  # block_time is in seconds; delay_buffer multiplier is to account for network delays
+    try:
+        mm_config = matchmaking.get_registration(chain_id)
+        interchain_network = mm_config["network"]  # returns: "btc" / "eth" / "bnb"
+        attr = network_attr[interchain_network]
+        broadcast_receipt_wait_time_l5 = int(attr["confirmations"] * attr["block_time"] * attr["delay_buffer"])  # in seconds
```

Comment:
We still need to take into account remaining_time_to_broadcast for the nodes. This duration won't be sufficient for the 3 hour wait of a btc L5 + confirmation time.

Reply:
Fixed, included broadcast interval.